nucypher/cli/characters/enrico.py

```diff
@@ -7,73 +9,81 @@
 from nucypher.cli.types import NETWORK_PORT
 
 
-@click.command()
-@click.argument('action')
+# Args (policy_encrypting_key)
+def _common_options(func):
```

Comment:
If there's no more descriptive name than "common options", it may be better to just define 
```
policy_encrypting_key_option = click.option('--policy-encrypting-key', help="Encrypting Public Key for Policy as hexadecimal string", type=click.STRING, required=True)
```

and then using it as decorator.

Reply:
✔️